src/test/java/org/apache/commons/io/input/QueueInputStreamTest.java

```diff
@@ -17,24 +17,29 @@
 package org.apache.commons.io.input;
 
 import static java.nio.charset.StandardCharsets.UTF_8;
+import static org.apache.commons.io.input.QueueInputStream.createBlockingQueueInputStream;
```

Comment:
@maxxedev 
Only use static imports for JUnit assertions, otherwise, it's too confusing as to who does what IMO.